webpack.config.js

```diff
@@ -1,5 +1,9 @@
 const path = require('path');
```

Comment:
- [ ] **Required:** Do take your time to reprocess the[ configuration of Webpack,](https://webpack.js.org/guides/getting-started/) it throws major errors that prevent the smooth running of your app.

**Hint:** Consider this 

```
const path = require("path");
const HtmlWebpackPlugin = require("html-webpack-plugin");

module.exports = {
   //more config
}
```

Reply:
I just add a title to the plugin.